internal/controller/volumereplicationgroup_controller.go

```diff
@@ -867,7 +867,7 @@ func (v *VRGInstance) separatePVCUsingPeerClassAndSC(peerClasses []ramendrv1alph
 		msg := fmt.Sprintf("peerClass matching storageClass %s not found for async PVC", storageClass.GetName())
 		v.updatePVCDataReadyCondition(pvc.Namespace, pvc.Name, VRGConditionReasonPeerClassNotFound, msg)
 
-		return fmt.Errorf(msg)
+		return fmt.Errorf("peerClass matching storageClass %s not found for async PVC", storageClass.GetName())
```

Comment:
A better fix is using errors.New(msg)

Reply:
`errors.New` only takes a string, and no parameters. In this instance the error has a specific SC name in it, hence sticking to `fmt.Errorf` with an explicit message.

Reply:
But previously we formatted msg once - so we could use errors.New(msg).

Reply:
Strangely the linter has no complait on doing this (i.e using errors.New on a non-constant...) changed it in required places.